rdmo/accounts/tests/test_views.py

```diff
@@ -39,20 +38,20 @@
 boolean_toggle = (True, False)
 
 
-@pytest.fixture(autouse=True, scope='function')
-def clear_url_cache_at_teardown():
+@pytest.fixture(autouse=True, scope='module')
```

Comment:
was this the only thing that effectively fixed the tests?

Reply:
yes, `scope='module'` calles the teardown after `yield` only after the module. The problem was that the urls with `settings.ACCOUNT = False` were still there when the tests proceeded to `projects`.

Reply:
alright, thanks!